manifests/params.pp

```diff
@@ -14,4 +14,19 @@
   $ttl               = false
   $packages          = ['corosync', 'pacemaker']
   $token             = 3000
+
+  case $::osfamily {
+    'RedHat': {
+      $set_votequorum = true
```

Comment:
This will cause the module to start failing on RedHat because there are no default quorum members. Why does this need to default to true? If it needs to be true, is there a default quorum_members value that makes sense, e.g. localhost?